userspace/libscap/scap_procs.c

```diff
@@ -941,6 +1007,11 @@ static int32_t scap_proc_add_from_proc(scap_t* handle, uint32_t tid, char* procd
 		return SCAP_FAILURE;
 	}
 
+	if (scap_proc_fill_overlayfs_root(handle, tinfo, dir_name) == SCAP_FAILURE)
+	{
+		// NOT AN ERROR; skip
```

Comment:
We do not consider an error if overlayfs cannot be scanned; containers will just miss users and groups informations.